rmgpy/cantherm/molpro.py

```diff
@@ -260,58 +260,89 @@ def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=1, sym
-        while line!='':
+        # Determine keywords to look for. Currently CCSD(T)-F12a, CCSD(T)-F12b,
+        # MRCI, MRCI+Davidson are supported
+        f12a, f12b, mrci = False, False, False
+        while line != '':
             if 'basis' in line.lower():
                 if 'vtz' in line.lower() or 'vdz' in line.lower():
-                    f12a=True
-                else: f12a=False
+                    f12a = True
+                    # MRCI could also have a vdz/vtz basis, so don't break
+                elif any(high_basis in line.lower() for high_basis in ['vqz', 'v5z', 'v6z', 'v7z', 'v8z']):
+                    f12b = True
+                    # MRCI could also have a v(4+)z basis, so don't break
+            elif 'mrci' in line.lower():
+                mrci = True
+                f12a, f12b = False, False
                 break
+            elif 'point group' in line.lower():
+                # we expect to already know the method by this point, so break if possible, but don't throw an error yet
+                if any([mrci, f12a, f12b]):
+                    break
             line=f.readline()
-        else: raise Exception('Could not find basis set in Molpro File')
-        #search for energy
-        E0=None
+        else:
+            raise ValueError('Could not determine type of calculation. Currently, CCSD(T)-F12a, CCSD(T)-F12b,'
+                             ' MRCI, MRCI+Davidson are supported')
+        # search for energy
+        E0 = None
         if f12a:
-            while line!='':
+            while line != '':
                 if ('RHF-UCCSD(T)-F12a energy' in line
                     or 'RHF-RCCSD(T)-F12a energy' in line
-                    or 'CCSD(T)-F12a total energy  ' in line):
-                    E0=float(line.split()[-1])
-                    break
+                        or 'CCSD(T)-F12a total energy  ' in line):
+                        E0 = float(line.split()[-1])
+                        break
                 if 'Electronic Energy at 0' in line:
-                    E0=float(line.split()[-2])
+                    E0 = float(line.split()[-2])
                     break
                 line=f.readline()
-        else:
-            while line!='':
+        elif f12b:
+            while line != '':
                 if ('RHF-UCCSD(T)-F12b energy' in line
                     or 'RHF-RCCSD(T)-F12b energy' in line
-                    or 'CCSD(T)-F12b total energy  ' in line):
-                    E0=float(line.split()[-1])
-                    break
+                        or 'CCSD(T)-F12b total energy  ' in line):
+                        E0 = float(line.split()[-1])
+                        break
                 if 'Electronic Energy at 0' in line:
-                    E0=float(line.split()[-2])
+                    E0 = float(line.split()[-2])
                     break
                 line=f.readline()
-
+        elif mrci and 'mrci+davidson' in model_chemistry.lower():
```

Comment:
See previous comment. Does Molpro always print the Davidson corrected energy if using MRCI? If not, I think we should remove `model_chemistry` in this function.